conf.json

```diff
@@ -0,0 +1,10 @@
+{ map: 
```

Comment:
Was to differentiate them, but I suppose it makes more sense to keep them consistent. Changed it to settings